parl/remote/test_utils.py

```diff
@@ -0,0 +1,42 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+from parameterized import parameterized, parameterized_class
+from unittest import mock
+
+# How to add new configuration: provide turples with two arguements, where the first one represents the function be be mocked, and the second one is the mocked return.
```

Comment:
turples -> tuples

Reply:
arguements -> arguments

Reply:
be be -> can be

Reply:
Sorry, I should check the spelling error before making a commitment.